rtCamp/privacy-sandbox-demos
# Removing under construction routes

## Description

This pull request removes the routes with under-construction content. This was feedback made on the weekly call on December 6th. The links should be only included when they are ready for the demo site. The folders for `fedcm`, private state tokes and related website sets were removed from the code base, including the under-construction page.

## Testing Instructions

1. Clone the repository and check out the `feature/removing-under-construction-routes` branch containing these changes.
2. Please make sure you have Node.js and npm installed.
3. Run `npm install `to install all dependencies.
5. Start the server using npm start.
6. You can navigate to the address: `http://localhost:8080/fedcm`, `http://localhost:8080/related-websites-sets`, and `http://localhost:8080/private-state-tokens`
7. All three URLs should point to the homepage
